zipline/finance/slippage.py

```diff
@@ -145,8 +146,8 @@ def process_order(self, trade_bar, order):
                 return
 
         return create_transaction(
-            trade_bar,
             order,
+            bar_data.current_dt,
```

Comment:
It looks like this is the only place where this needs the `bar_data` object now.  Could we just pass the dt so that this function just operates on plain old data?